src/Mod/TechDraw/Gui/CommandExtensionPack.cpp

```diff
@@ -1446,7 +1446,7 @@ void CmdTechDrawExtensionPositionSectionView::activated(int iMsg) {
             QObject::tr("Selection is empty"));
         return;
     }
-    float xPos, yPos;
+    float xPos=NAN, yPos= NAN;
```

Comment:
If we're going to initialize these to NaN, shouldn't we also check for it before assigning, below? It looks like there is indeed a branch where it's possible they aren't set, and I'm not sure it's a good idea to let this code deliver NaN as a result for the sectionView position.

Reply:
I also think it's better to initialize them with 0.0 instead.